dictionaries/companies/src/companies.txt

```diff
@@ -275,6 +275,7 @@ Camerfirma
 Cameron International Corporation
 Campbell Soup
 Campbell Soup Company
+Caniuse
```

Comment:
`Can I Use` is more of a service than a company. Adding it to the `npm` dictionary makes a lot of sense. Also adding it to [web services](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/software-terms/src/software-services.txt) would be good.